listeners/websocket.js

```diff
@@ -0,0 +1,31 @@
+var WebSocketClient = require('websocket').client;
+var deviceInfo = require('./../data/device');
+
+var init = exports.init = function(conf, logger) {
+    var client = new WebSocketClient();
+
+    client.on('connectFailed', function() {
+        logger.error("Websocket cannot connect.");
+        setTimeout(function() {
+            init(conf, logger);
+        }, parseInt(conf.connector.ws.retryTime));
+    });
+    client.on('connect', function(connection){
+        logger.info('Websocket listener started on port: ' + conf.connector.ws.port);
```

Comment:
this message is misguiding. You've created a client and do not accept incoming messages on this port. Try instead something like 'Connected to websocket server %h:%p'